src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -567,6 +572,19 @@ private async Task PackageSourcesChangedAsync(IReadOnlyCollection<PackageSourceC
             }
         }
 
+        private async Task IsCentralPackageManagementEnabledAsync(CancellationToken cancellationToken)
+        {
+            await NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(async delegate
+            {
+                // Go off the UI thread to perform non-UI operations
+                await TaskScheduler.Default;
+                foreach (IProjectContextInfo project in Model.Context.Projects)
+                {
+                    _detailModel.IsCentralPackageManagementEnabled = await project.IsCentralPackageManagementEnabledAsync(Model.Context.ServiceBroker, cancellationToken);
```

Comment:
If there are more than 1 projects, then `_detailModel.IsCentralPackageManagementEnable` going to have value from last project evaluation. 
Is it correct logic? Should it be aggregate of all projects or just last one?

Reply:
Maybe we should terminate the loop when `IsCentralPackageManagementEnabledAsync` method returns for any project.

Reply:
You're right @erdembayar, I have some logic before calling this function and this should only be run in Project level, but that makes this function non reusable, I'll modify it to get the aggregate even if it is just one project